commands/applications.go

```diff
@@ -21,9 +21,9 @@ func NewApplications(network *net.Network) (cmd *Applications) {
 
 func (cmd *Applications) Metadata() command_metadata.CommandMetadata {
 	return command_metadata.CommandMetadata{
-		Name:        "applications",
+		Name:        "application",
 		Description: "Show the status and location of running applications",
-		Usage:       "BROOKLYN_NAME applications",
```

Comment:
FYI there is a neat little trick going on here: `BROOKLYN_NAME` is automatically substituted with the binary's name in the help text, so recommend you keep that bit.